nutkit/protocol/requests.py

```diff
@@ -195,6 +195,29 @@ def __init__(self, requestId, addresses):
         self.addresses = addresses
 
 
+class BookmarksSupplierCompleted:
+    """
+    Results of a bookmark manager bookmark supplier call.
+
+    Pushes bookmarks for a given database to the Bookmark Manager.
+    """
+
+    def __init__(self, requestId, bookmarks):
+        self.requestId = requestId
+        self.bookmarks = bookmarks
+
+
+class BookmarksConsumerCompleted:
+    """
+    Results of a bookmark manager bookmarks consumer call.
```

Comment:
```suggestion
    Results of a bookmark manager's bookmarks consumer call.
```
that's a bit easier on the eyes ^^